DigitalLearningSolutions.Web/Controllers/LearningPortalController/FilteredResults.cs

```diff
@@ -1,126 +0,0 @@
-                filteredToken = accessToken.Jwt_access_token.ToString();
-                CookieOptions cookieOptions = new CookieOptions();
-                cookieOptions.Expires = new DateTimeOffset(DateTime.Now.AddMinutes(15));
-                Response.Cookies.Append("filtered-" + candidateNum, filteredToken, cookieOptions);
-            }
-            return filteredToken;
-        }
-        [Route("/LearningPortal/SelfAssessment/{selfAssessmentId:int}/Filtered/Results")]
-        public async Task<IActionResult> SelfAssessmentFilteredResults(int selfAssessmentId)
-        {
-            var candidateID = User.GetCandidateIdKnownNotNull();
-            string destUrl = "/LearningPortal/SelfAssessment/" + selfAssessmentId.ToString() + "/Filtered/Dashboard";
-            selfAssessmentService.SetBookmark(selfAssessmentId, candidateID, destUrl);
-            var assessment = selfAssessmentService.GetSelfAssessmentForCandidateById(candidateID, selfAssessmentId);
-            if (assessment == null)
-            {
-                logger.LogWarning($"Attempt to display self assessment Filtered API results for candidate {candidateID} with no self assessment");
-                return RedirectToAction("StatusCode", "LearningSolutions", new { code = 403 });
-            }
-            selfAssessmentService.UpdateLastAccessed(assessment.Id, candidateID);
-            var filteredToken = await GetFilteredToken();
-            var profile = selfAssessmentService.GetFilteredProfileForCandidateById(selfAssessmentId, candidateID);
-            var goals = selfAssessmentService.GetFilteredGoalsForCandidateId(selfAssessmentId, candidateID).ToList();
-            var response = await filteredApiHelperService.UpdateProfileAndGoals(filteredToken, profile, goals);
-            var favouritePlayList = await filteredApiHelperService.GetPlayList<PlayList>(filteredToken, "playlist.FetchFavouritePlaylist", null);
-            selfAssessmentService.SetSubmittedDateNow(selfAssessmentId, candidateID);
-            selfAssessmentService.SetUpdatedFlag(selfAssessmentId, candidateID, false);
-            var model = new SelfAssessmentFilteredResultsViewModel()
-             {
-                 SelfAssessment = assessment,
-                 CompetencyPlayLists = await filteredApiHelperService.GetPlayListsPoll<IEnumerable<PlayList>>(filteredToken, "playlist.FetchCompetencyPlaylists"),
-                 RecommendedPlayLists = await filteredApiHelperService.GetPlayListsPoll<IEnumerable<PlayList>>(filteredToken, "playlist.FetchNexRexPlaylists"),
-                 FavouritePlayList = favouritePlayList
-            };
-            return View("SelfAssessments/FilteredMgp/FilteredResults", model);
-        }
-        [Route("/LearningPortal/SelfAssessment/{selfAssessmentId:int}/Filtered/Dashboard")]
-        public async Task<IActionResult> FilteredRecommendations(int selfAssessmentId)

```

Comment:
There are a few places that still attempt to link to these old actions that will probably be broken if accessing the relevant pages now with signposting disabled. These are mostly in back links.
Searching for "FilteredRecommendations" should bring up a few locations to bring back to.

Reply:
I think I've caught all these now.

Reply:
Looks like there are still a couple on Playlist.cshtml.